loader/src/Loader.cc

```diff
@@ -368,7 +368,7 @@ namespace ignition
 #endif
 
       void *dlHandle = dlopen(_pathToLibrary.c_str(),
-                              RTLD_NOLOAD | RTLD_LAZY | RTLD_LOCAL);
+                              RTLD_NOLOAD | RTLD_LAZY | RTLD_GLOBAL);
```

Comment:
What do you think, @mxgrey , are we trying to do something that `ign-plugin` wasn't meant to do, or is this a reasonable change?

See https://github.com/ignitionrobotics/ign-sensors/pull/38#issuecomment-678123125 for the reason why this change is needed.